TRAMS-API/Mapping/GetAcademiesD365ModelToGetAcademiesModelMapper.cs

```diff
@@ -23,7 +23,7 @@ public GetAcademiesModel Map(GetAcademiesD365Model input)
                 LocalAuthorityNumber = input.LocalAuthorityNumber,
                 OfstedInspectionDate = input.OfstedInspectionDate,
                 OfstedRating = input.OftstedRating,
-                ParentTrustId = input.ParentTrustId,
+                ParentTrustId = input.ParentTrustId.Value,
```

Comment:
Warning: input.ParentTrustId is nullable.  Suggest making ParentTrustId on the output model nullable or handling a potential null input value.

Reply:
Model validation requires the Guid to be set before the mapping is done. Should that cover it ?